modules/rest-api/openapi/openapi.yaml

```diff
@@ -833,11 +797,9 @@ components:
         clusterName:
           type: string
           description: The name of the cluster.
-        authenticationConfig:
-          allOf:
-          - $ref: '#/components/schemas/AuthenticationConfig'
-          - type: object
-            description: Authentication configuration.
+        clusterConfiguration:
+          type: string
+          description: Cluster configuration.
```

Comment:
Could you adjust the description? I think it would be useful to mention the HOCON format of the provided string.